.gitignore

```diff
@@ -0,0 +1,2 @@
+test.md
+node_modules
```

Comment:
- [ ] Great job adding test.md to .gitignore, however please add a forward slash `/` in front of node_modules because it node_modules is a directory.

### EXAMPLE
```
test.md
node_modules/
```